### PR TITLE
Fix reflines

### LINKED ISSUE #57: Hide reflines in separate view
In "separate" view, the three voices are at different y-levels in the focus view, so the reflines are inaccurate.

Please consider hiding them (more difficult would be to translate them appropriately).

### PR DESCRIPTION
**Pull request by curran**
Renders reference lines properly for all states.

Will close #57

### CODE REVIEW

**Comment by curran** [comment]
Here's what we've got - I believe all the bases are covered now.

Initial load - only the bottom layer has the ref lines.

image

Separated view - each voice has a properly scaled set of ref lines. The transitions need work, but the rendering is correct.

image

Separated with hilite - reflines are shown on the hilited voice only.

image

@seemantk I believe this is ready for merge.

**Comment by seemantk** [comment]
@curran, thanks!  this looks great.  I'm seeing this error in the console: reflines

**Comment by curran** [comment]
Oh my! So it does. Sorry about that, investigating.

**Comment by curran** [comment]
@seemantk Fixed.

File: js/notescanvas.js [hunk 1]
```diff
@@ -41,11 +41,7 @@ function NotesCanvas() {
             .map(function(d) { return d.pitch; })
             .concat(reflinesPitches)
         ;
-        domain.y = d3.range(
-              d3.min(pitches)
-            , d3.max(pitches)
-          )
-        ;
+        domain.y = d3.range.apply(null, d3.extent(pitches));
```
Comment (seemantk) [inline js/notescanvas.js:44]: I like this a lot.  TIL...

**Pull request merged**